src/ui/component/fileViewer/view.jsx

```diff
@@ -65,6 +65,10 @@ export default function FileViewer(props: Props) {
   const previousIsReadyToPlay = usePrevious(isReadyToPlay);
   const isNewView = uri && previousUri !== uri && isPlaying;
   const wasntReadyButNowItIs = isReadyToPlay && !previousIsReadyToPlay;
+  const logViewAndClaimRewards = async (uri, timeToStart) => {
+    await triggerAnalyticsView(uri, timeToStart);
```

Comment:
This won't work because this function doesn't return a promise after the view event is complete
https://github.com/lbryio/lbry-desktop/blob/master/src/ui/analytics.js#L69